docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,479 @@
+
+2. Install Teleport on each instance.
+
+   <Admonition type="warning" title="Warning">
+    The examples below may include the use of the `sudo` keyword to make following each step easier when creating resources from scratch. However, we generally discourage the use of `sudo` in real-world production environments per the *Principle of Least Privilege* (POLP). 
+   </Admonition>
+
+   <Tabs>
+     <TabItem label="Amazon Linux 2/RHEL (RPM)">
+      ```bash
+      sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+      sudo yum install teleport
+
+      # Optional:  Using DNF on newer distributions
+      # $ sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+      # $ sudo dnf install teleport
+      ```
+     </TabItem>
+
+     <TabItem label="Debian/Ubuntu (DEB)">
+      ```bash
+      curl https://deb.releases.teleport.dev/teleport-pubkey.asc | sudo apt-key add -
+      sudo add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
+      sudo apt-get update
+      sudo apt-get install teleport
+      ```
+     </TabItem>
+
+     <TabItem label="Linux">
+      ```bash
+      curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+      tar -xzf teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+      cd teleport
+      sudo ./install
+      ```
+     </TabItem>
+
+     <TabItem label="ARMv7 (32-bit)">
+      ```bash
+      curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm-bin.tar.gz
+      tar -xzf teleport-v(=teleport.version=)-linux-arm-bin.tar.gz
+      cd teleport
+      sudo ./install
+      ```
+     </TabItem>
+
+     <TabItem label="ARMv8 (64-bit)">
+      ```bash
+      curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
+      tar -xzf teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
+      cd teleport
+      sudo ./install
+      ```
+     </TabItem>
+   </Tabs>
+
+3. Configure Teleport on the *Bastion Host*.
+
+   Teleport will now automatically acquire an ACME-compliant TLS certificate. 
+
+   ```bash
+   # Configure Teleport with TLS certs
+   sudo teleport configure \
+     --acme --acme-email=your_email@example.com \
+     --cluster-name=tele.example.com \
+     -o file
+   ```
+
+   Run the command above on `tele.example.com`.
+
+4. Launch your *Bastion Host* by running:
+
+   ```bash
+   # Launch Teleport
+   sudo teleport start
+   ```
+
+   Next, we'll create a *join token* to add and start the second Node.
+
+## Step 2/5. Add a Node to the cluster
+
+1. Create a *join token* to add the second Node to the `tele.example.com` cluster. In a second, *Bastion Host*, terminal run the following command:
+
+   <Tabs>
+     <TabItem label="Create a token">
+      ```bash
+      # Let's save the token to a file
+      sudo tctl tokens add --type=node > token.file
```

Comment:
As described, this command will not work, however this will:

```
$ tctl tokens add --type=node | grep -oP '(?<=token:\s).*' > token.file
```

I recommend testing the docs by following them command by command and making sure the copied commands work.

Reply:
Thanks for the recommendation @klizhentas - I did in fact go through and test these per your suggestion - I think the code snippet above was copied directly from your earlier comment (far above) and tested:

![Screen Shot 2021-05-12 at 7 23 44 PM](https://user-images.githubusercontent.com/82530006/118060135-b11e1f00-b357-11eb-8a94-e53faaa16eaa.png)

Am I missing something?

Reply:
This part works, but if you try to start a node with the next command, teleport start --token=./token will not work, because the token file contains full text, but not the token.

Reply:
Yeah for the example above I switched the name to `token` for ease. I could have used `token.file` instead (as you well know) with the same result. The CLI is indifferent to the suffix only the supplied token path.